examples/examples.rs

```diff
@@ -82,37 +80,26 @@ fn examples() -> Result<()> {
     // This API handles variadics using Heterogeneous Lists.  This is one way to
     // call a function with multiple parameters:
 
-    print.call::<_, ()>(
-        hlist!["hello", "again", "from", "rust"],
-    )?;
+    print.call::<_, ()>(("hello", "again", "from", "rust"))?;
```

Comment:
This is definitely an improvement! The double `((` is a limitation of Rust which I don't think we should work around.

Actually, Rust closures work in much the same way, their arguments are packed into a tuple internally.